frontend/lib/src/components/elements/Metric/Metric.tsx

```diff
@@ -42,85 +41,74 @@ export interface MetricProps {
 export default function Metric({
   element,
 }: Readonly<MetricProps>): ReactElement {
-  const { colors }: EmotionTheme = useTheme()
-  const { MetricColor, MetricDirection } = MetricProto
+  const { MetricDirection } = MetricProto
+  const {
+    body,
+    label,
+    delta,
+    direction,
+    color,
+    labelVisibility,
+    help,
+    border,
+  } = element
 
-  let direction: any = null
-  let color = ""
+  let metricDirection: any = null
 
-  switch (element.color) {
-    case MetricColor.RED:
-      color = colors.metricNegativeDeltaColor
-      break
-    case MetricColor.GREEN:
-      color = colors.metricPositiveDeltaColor
-      break
-    // this must be grey
-    default:
-      color = colors.metricNeutralDeltaColor
-      break
-  }
-
-  switch (element.direction) {
+  switch (direction) {
     case MetricDirection.DOWN:
-      direction = ArrowDownward
+      metricDirection = ArrowDownward
       break
     case MetricDirection.UP:
-      direction = ArrowUpward
+      metricDirection = ArrowUpward
       break
     // this must be none
     default:
-      direction = null
+      metricDirection = null
```

Comment:
This does not seem to be required because you already initialize `metricDirection` with `null` in [line 56](https://github.com/streamlit/streamlit/pull/9928/files#diff-aa41fea932118f5233dbba41b388ea4465b8cd931325ba1e7297bb5c9affb35eR56)